workbench/views.py

```diff
@@ -80,14 +79,20 @@ def show_scenario(request, scenario_id, view_name='student_view', template='work
     })
 
 
-def user_list(request):
+def user_list(_request):
```

Comment:
To clarify, this change actually changes the signature of this function if the caller expects to pass this argument as a keyword: `user_list(request=foo)`. We should only make this change if we're sure that no caller does this. If we think that a caller might do this, then it would be better to just use a `pylint: disable` here.